src/app/pipes/sorting.pipe.ts

```diff
@@ -116,6 +118,32 @@ export class SortingPipe implements PipeTransform {
       }
     }
 
+    if (property === 'folderSize') {
+
+      // want non-folders to be considered "less than" a folder so give negative value by default.
+      var numCheck = /^\d+$/;
+      var xDisplay = -Infinity;
+      var yDisplay = -Infinity;
+
+      // numcheck because a folder will have num videos for this value, videos will have mb or gb characters for size
+      if(numCheck.test(x.fileSizeDisplay)){
```

Comment:
I wonder if there's a simpler way -- to check 
```js
if (x.cleanName === "*FOLDER*") { 
```
🤔 
I feel like co-opting a property rather than creating a new one for displaying folders may not have been the best move, but I'm still unsure 🤷 😅 

As an example, I check if something is a folder using the above approach: https://github.com/whyboris/Video-Hub-App/blob/main/src/app/components/views/thumbnail/thumbnail.component.html#L10